python/build-wheel-manylinux1.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cat << EOF > "/usr/bin/nproc"
+#!/bin/bash
+echo 1
+EOF
+chmod +x /usr/bin/nproc
+
+mkdir .whl
+for PYTHON in cp27-cp27mu cp33-cp33m cp34-cp34m cp35-cp35m cp36-cp36m; do
+  rm -rf *
+  git checkout *
+  pushd python
+  /opt/python/${PYTHON}/bin/pip install numpy
```

Comment:
You want to fix this to an older numpy version. This will be the oldest numpy that you can support, From experiences with Apache Arrow, I suggest 1.10.4